src/generic/sub-header/SubHeader.scss

```diff
@@ -16,6 +16,11 @@
     display: block;
     color: $text-color-base;
   }
+
+  .sub-header-title-actions {
+    margin-top: .5em;
+    justify-content: flex-start;
+  }
```

Comment:
I think this should be equivalent to `mt-2 justify-content-start`

Reply:
Updated here: https://github.com/openedx/frontend-app-course-authoring/pull/723/commits/80b736eb44386aac0fa85bb8d3cee2523471f09b